manifold_test.go

```diff
@@ -39,13 +39,22 @@ func TestConfig_WithAPIToken(t *testing.T) {
 	hct := &headerCheckTransport{}
 	http.DefaultTransport = hct
 
-	token := os.Getenv("MANIFOLD_API_TOKEN")
-
 	t.Run("without extra configuration", func(t *testing.T) {
 		c := manifold.New()
 
 		hct.reset()
-		hct.expectHeaderEquals(t, "Authorization", fmt.Sprintf("Bearer %s", token))
+		hct.expectHeaderEquals(t, "Authorization", "")
+
+		c.Plans.List(context.Background(), nil)
+	})
+
+	t.Run("with env var", func(t *testing.T) {
+		os.Setenv("MANIFOLD_API_TOKEN", "s3cr3t")
```

Comment:
It'd do a defer here to set it back to the original value afterwards. (so read the env, set a new one, and defer to the initial one)